ubuntudesign/snapcraft.io
# Add ./import-docs.sh

A script for pulling in docs from canonicalltd/snappy-docs, and making them available at /docs.

QA
--

First make sure you have Python and Pip 3.5 or greater (`pip3 --version`). Then install `python-frontmatter`:

```
pip3 install python-frontmatter
```

Run docs script, before building the site:

```
./import-docs.sh
npm install
node_modules/gulp/bin/gulp.js
```

Visit http://127.0.0.1:3000/docs and check it looks okay.
